wit/store.wit

```diff
@@ -7,22 +7,65 @@
 /// ensuring compatibility between different key-value stores. Note: the clients will be expecting
 /// serialization/deserialization overhead to be handled by the key-value store. The value could be
 /// a serialized object from JSON, HTML or vendor-specific data types like AWS S3 objects.
+///
+/// ## Consistency and Cache Coherency
```

Comment:
I think this section is mostly discussing the topic of data consistency. Cache coherency was never introduced or mentioned, as far as I can tell, no?

Reply:
Good catch.  I originally said more about caching, then removed it.  All that's left is the "due to e.g. cache effects and/or replication lag" part.  Anyway, I'll rename the heading.